### PR TITLE
Add more detail to validator gRPC metrics.

### PR DESCRIPTION
**Pull request by aschran**

### CODE REVIEW

File: crates/sui-node/src/metrics.rs [hunk 1]
```diff
@@ -187,8 +188,13 @@ pub fn start_metrics_push_task(config: &sui_config::NodeConfig, registry: Regist
 pub struct GrpcMetrics {
     inflight_grpc: IntGaugeVec,
     grpc_requests: IntCounterVec,
+    grpc_request_latency: HistogramVec,
 }
 
+const LATENCY_SEC_BUCKETS: &[f64] = &[
```
Comment (dmitri-perelman) [inline crates/sui-node/src/metrics.rs:194]: nit: we could probably use a list similar to the one used in NW primary / worker latencies.

**Review by dmitri-perelman — APPROVED** [review]

**Review by bmwill — APPROVED** [review]

File: crates/sui-node/src/metrics.rs (hunk 1 above)
Comment (aschran) [reply crates/sui-node/src/metrics.rs:194]: I actually copy-pasted this directly from the NW one :)

want me to add a TODO to dedupe it somewhere?

Comment (dmitri-perelman) [reply crates/sui-node/src/metrics.rs:194]: Hehe, we probably have several places with these in NW, I was referring to the (more comprehensive) one used by the primary and worker metrics: https://github.com/MystenLabs/sui/blob/main/narwhal/primary/src/metrics.rs#L17
We can upgrade it later if we see the need.

Comment (aschran) [reply crates/sui-node/src/metrics.rs:194]: Ohhh gotcha! Yeah I cribbed the ones from anemo metrics, I see what you mean about the more detailed ones. Sounds good.

**Pull request merged**